src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/Extensions/SAP_Internal_Stats_KeyFigure+Formatting.swift

```diff
@@ -5,18 +5,19 @@
 import UIKit
 
 extension SAP_Internal_Stats_KeyFigure {
+
+	// MARK: - Internal
+
 	var formattedValue: String? {
 		let decimals = max(0, Int(self.decimals))
-		let numberFormatter = NumberFormatter()
-		numberFormatter.numberStyle = .decimal
+		let numberFormatter = Self.decimalNumberFormatter
 		numberFormatter.minimumFractionDigits = Int(decimals)
 		numberFormatter.maximumFractionDigits = Int(decimals)
```

Comment:
Moving this number formatter to a static let makes it not thread safe anymore if multiple KeyFigures are asked for their `formattedValue` in parallel, as the same shared object is reconfigured now. Unlikely that there are any practical implications here, but I want to mention it.